core/src/main/scala/org/apache/spark/scheduler/SparkListener.scala

```diff
@@ -160,7 +160,6 @@ case class SparkListenerApplicationEnd(time: Long) extends SparkListenerEvent
 
 /**
  * An internal class that describes the metadata of an event log.
- * This event is not meant to be posted to listeners downstream.
```

Comment:
This was only for metadata info, so when this was written it was just not meant to be consumed but now we can reuse it for this case